scripts/screeps-runtime-monitor.py

```
@@ -2075,6 +2094,95 @@ def runtime_build_action_result_counts(room: dict[str, Any] | None) -> dict[str,
+def runtime_summary_evidence_tick(room: dict[str, Any] | None) -> int | None:
+    tick = runtime_summary_room_tick(room)
+    if tick is not None:
+        return tick
+    if isinstance(room, dict):
+        return tick_number(room.get("runtimeSummaryTick"))
+    return None
+
+
+def runtime_summary_evidence_is_recent(
+    room: dict[str, Any] | None,
+    current_tick_value: Any,
```

Ruff (0.15.15)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `current_tick_value`

(ANN401)